src/components/TutorialMetadata.tsx

```diff
@@ -1,20 +1,19 @@
 import React from "react"
-import { useI18next } from "gatsby-plugin-react-i18next"
 import { Badge, Box, Flex, HStack } from "@chakra-ui/react"
 
 import Emoji from "./Emoji"
-import CopyToClipboard from "./CopyToClipboard"
 import InlineLink from "./Link"
 import TutorialTags from "./TutorialTags"
-import Translation from "./Translation"
-import Text from "./OldText"
 
-import { Lang } from "../utils/languages"
-import { getLocaleTimestamp } from "../utils/time"
-import { TranslationKey } from "../utils/translations"
+// TODO: Implement after intl
+// import Translation from "./Translation"
+// import { Lang } from "../utils/languages"
+// import { getLocaleTimestamp } from "../utils/time"
+import type { TranslationKey } from "@/lib/types"
 
 export interface IProps {
```

Comment:
Nice to have: implement new TS conventions.
```suggestion
export type TutorialMetadataProps = {
```